Rxjs with @cycle/time v0.12.0 doesn't build

Just a heads up, I saw these errors while trying to integrate the latest cycle/time into a project:
```
ERROR in ./node_modules/@cycle/time/rxjs.js
Module not found: Error: Can't resolve './src/mock-time-source' in '/Users/ntilwalli/dev/spotlight/apps/candle/node_modules/@cycle/time'
 @ ./node_modules/@cycle/time/rxjs.js 7:25-58
 @ ./web/static/ts/main.ts

ERROR in ./node_modules/@cycle/time/rxjs.js
Module not found: Error: Can't resolve './src/time-driver' in '/Users/ntilwalli/dev/spotlight/apps/candle/node_modules/@cycle/time'
 @ ./node_modules/@cycle/time/rxjs.js 8:20-48
 @ ./web/static/ts/main.ts

ERROR in [at-loader] ./node_modules/@cycle/time/rxjs.d.ts:3:23 
    TS2307: Cannot find module './src/animation-frames'.

ERROR in [at-loader] ./node_modules/@cycle/time/rxjs.d.ts:4:42 
    TS2307: Cannot find module './src/types'.
```

--- a/time/most.ts
+++ b/time/most.ts
@@ -2,10 +2,10 @@ import * as most from 'most';
 import {Stream} from 'most';
 import {setAdapt} from '@cycle/run/lib/adapt';
 
-import {mockTimeSource as mockTimeSourceUntyped} from './src/mock-time-source';
-import {timeDriver as timeDriverUntyped} from './src/time-driver';
-import {Frame} from './src/animation-frames';
-import {Comparator, OperatorArgs} from './src/types';
+import {mockTimeSource as mockTimeSourceUntyped} from './lib/cjs/src/mock-time-source';
+import {timeDriver as timeDriverUntyped} from './lib/cjs/src/time-driver';
+import {Frame} from './lib/cjs/src/animation-frames';
+import {Comparator, OperatorArgs} from './lib/cjs/src/types';
 
 setAdapt(stream => most.from(stream as any));
 

--- a/time/package.json
+++ b/time/package.json
@@ -59,6 +59,7 @@
     "test-ci": "npm run test",
     "test-docs": "markdown-doctest",
     "browserify": "../node_modules/.bin/browserify lib/cjs/index.js --global-transform=browserify-shim --standalone CycleTime --exclude xstream --outfile dist/cycle-time.js",
-    "minify": "node ../.scripts/minify.js dist/cycle-time.js dist/cycle-time.min.js"
+    "minify": "node ../.scripts/minify.js dist/cycle-time.js dist/cycle-time.min.js",
+    "postlib": "tsc rxjs.ts most.ts --declaration --lib DOM,ES5,ES6"
   }
 }

--- a/time/rxjs.ts
+++ b/time/rxjs.ts
@@ -3,10 +3,10 @@ import {Observable} from 'rxjs/Observable';
 import 'rxjs/add/observable/from';
 import {setAdapt} from '@cycle/run/lib/adapt';
 
-import {mockTimeSource as mockTimeSourceUntyped} from './src/mock-time-source';
-import {timeDriver as timeDriverUntyped} from './src/time-driver';
-import {Frame} from './src/animation-frames';
-import {Comparator, OperatorArgs} from './src/types';
+import {mockTimeSource as mockTimeSourceUntyped} from './lib/cjs/src/mock-time-source';
+import {timeDriver as timeDriverUntyped} from './lib/cjs/src/time-driver';
+import {Frame} from './lib/cjs/src/animation-frames';
+import {Comparator, OperatorArgs} from './lib/cjs/src/types';
 
 setAdapt(stream => Observable.from(stream));
 

--- a/time/tsconfig.json
+++ b/time/tsconfig.json
@@ -12,8 +12,6 @@
   },
   "files": [
     "index.ts",
-    "most.ts",
-    "rxjs.ts",
     "custom-typings.d.ts"
   ]
 }

